.github/pull_request_template.md

```diff
@@ -13,4 +13,4 @@
 
 * [ ] Does this PR change the input/output behaviour of a cryptographic algorithm (i.e., does it change known answer test values)?  (If so, a version bump will be required from *x.y.z* to *x.(y+1).0*.)
 
-<!-- Once your pull request is ready for review and passing continuous integration tests, please convert from a draft PR to a normal PR, and request a review from one of the OQS core team members. -->
\ No newline at end of file
+<!-- Once your pull request is ready for review and passing continuous integration tests, please convert from a draft PR to a normal PR, and request a review from one of the OQS core team members. -->
```

Comment:
`OQS core team members` -> `mlkem-c-aarch64 maintainers`?

Reply:
I think that is unrelated to my changes? 
We need to clean up:
- https://github.com/pq-code-package/mlkem-c-aarch64/blob/main/.github/ISSUE_TEMPLATE/bug_report.md
- https://github.com/pq-code-package/mlkem-c-aarch64/blob/main/.github/ISSUE_TEMPLATE/feature_request.md
- https://github.com/pq-code-package/mlkem-c-aarch64/blob/main/CONTRIBUTING.md
- https://github.com/pq-code-package/mlkem-c-aarch64/blob/main/GOVERNANCE.md
- https://github.com/pq-code-package/mlkem-c-aarch64/blob/main/SECURITY.md
- https://github.com/pq-code-package/mlkem-c-aarch64/blob/main/SUPPORT.md

Reply:
Let's discuss in a seperate PR: https://github.com/pq-code-package/mlkem-c-aarch64/pull/41